hadoop-ozone/dist/src/main/compose/ozone-mr/hadoop27/docker-config

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-CORE-SITE.xml_fs.AbstractFileSystem.o3fs.impl=org.apache.hadoop.fs.ozone.OzFs
```

Comment:
What's wrong with `OzFS`? Based on my understanding `ozonefs-hadoop2` project includes an `OzFs` implementation which is the same as `BasicOzFs`.

Is there any motivation behind this change?

Reply:
> Is there any motivation behind this change?

Yes, with `OzFs` I ran into `Invalid path name Wrong FS` (HDDS-3482).

Reply:
Got it. In that case it's my fault, the fix should be added to the hadoop2 specific `OzFs`. 

One important point of the new structure is that we don't need to use different classes (like `BasicOzFs`) we can use the same FS names all the time.

If it's not a big deal, can you please add the workaround instead of changing the name of the AbstractFileSystem implementation? 

```java
diff --git hadoop-ozone/ozonefs-hadoop2/src/main/java/org/apache/hadoop/fs/ozone/OzFs.java hadoop-ozone/ozonefs-hadoop2/src/main/java/org/apache/hadoop/fs/ozone/OzFs.java
index 914832e2c..b5012f95c 100644
--- hadoop-ozone/ozonefs-hadoop2/src/main/java/org/apache/hadoop/fs/ozone/OzFs.java
+++ hadoop-ozone/ozonefs-hadoop2/src/main/java/org/apache/hadoop/fs/ozone/OzFs.java
@@ -40,4 +40,9 @@ public OzFs(URI theUri, Configuration conf)
     super(theUri, new OzoneFileSystem(), conf,
         OzoneConsts.OZONE_URI_SCHEME, false);
   }
+
+  @Override
+  public int getUriDefaultPort() {
+    return -1;
+  }
 }
```

Tested and worked well.

Reply:
Thanks for the hint, workaround added.